sdk/react/src/initializeDVCClient.ts

```diff
@@ -1,18 +1,18 @@
-import { DVCOptions, initialize } from '@devcycle/devcycle-js-sdk'
-import type { DVCUser, DVCClient } from '@devcycle/devcycle-js-sdk'
+import { DevCycleOptions, initializeDevCycle } from '@devcycle/devcycle-js-sdk'
+import type { DevCycleUser, DevCycleClient } from '@devcycle/devcycle-js-sdk'
 
 const initializeDVCClient = (
     sdkKey: string,
-    user: DVCUser = { isAnonymous: true },
-    options?: DVCOptions,
-): DVCClient => {
+    user: DevCycleUser = { isAnonymous: true },
+    options?: DevCycleOptions,
+): DevCycleClient => {
     if (options?.deferInitialization) {
-        return initialize(sdkKey, {
+        return initializeDevCycle(sdkKey, {
             ...options,
             deferInitialization: true, // make typescript happy
         })
     }
-    return initialize(sdkKey, user, options)
+    return initializeDevCycle(sdkKey, user, options)
 }
 
 export default initializeDVCClient
```

Comment:
do we need to change this to be `initializeDevCycleClient` and deprecate `initializeDVCClient`

Reply:
yup missed this one.